Razorpay.nuspec

```diff
@@ -15,10 +15,12 @@
     <tags>Razorpay</tags>
     <dependencies>
       <dependency id="Newtonsoft.Json" version="11.0.2" />
+	  <dependency id="Newtonsoft.Json" version="13.0.3" />
```

Comment:
Fix the Indentation.

Reply:
I believe there is a better way for handling multiple versions of the same assembly. 
Refer this - https://learn.microsoft.com/en-us/dotnet/framework/configure-apps/redirect-assembly-versions

Reply:
Added 
```
<dependency id="Newtonsoft.Json" version="[11.0.2,13.0.3)" />
```

![Screenshot 2023-08-24 191129](https://github.com/razorpay/razorpay-dot-net/assets/89454448/853526f5-c42d-43b8-80ed-5305a4ee8fea)